assets/messages.yml

```
@@ -352,7 +352,7 @@ messages:
         *Thank you for your report!*
         We're tracking this issue in *%s%*, so this ticket is being resolved and linked as a *duplicate*.
 
-        That ticket has already been resolved as Fixed. The fix will arrive in a future version or is already included in the latest development version of the game; you can check the Fix Version/s field in that ticket to learn more.
+        That ticket has already been resolved as Fixed. Please check the Fix Version/s field in that ticket to see what version(s) the fix applies to.
```

> what version(s) the fix applies to

That might be a little bit misleading because the field only specifies in which version the bug was fixed (at least for Java edition). Maybe the following would be better?
```suggestion
        That ticket has already been resolved as Fixed. Please check the Fix Version/s field in that ticket to see in which version this behavior was or will be fixed.
```